### ISSUE TITLE
backup does not work

### ISSUE THREAD

**Issue by itmsergeev** [Altinity/clickhouse-backup#33]
./clickhouse-backup -config config.yml  create
2019/06/06 16:11:51 Create backup '2019-06-06T13-11-51'
2019/06/06 16:11:51 Freeze 'pf.blogger_stat'
2019/06/06 16:11:51   partition '201905'
2019/06/06 16:11:51   partition '201906'
2019/06/06 16:11:51 Freeze 'pf.migration'
2019/06/06 16:11:51   partition '197001'
2019/06/06 16:11:52 can't freeze partition '197001' on 'pf.migration' with: code: 368, message: std::bad_typeid

root@pclickhouse:/lib65/clickhouse-backup# clickhouse-server -V
ClickHouse server version 19.5.3.8 (official build).

From console - it works:

localhost :) ALTER TABLE pf.migration FREEZE PARTITION ''

ALTER TABLE pf.migration
    FREEZE PARTITION ''

Ok.

0 rows in set. Elapsed: 0.091 sec.

**Comment by AlexAkulov** [comment]
Hello!
What version of clickhouse-backup do you use?
Please, write output of these commands:
```
select create_table_query,partition_key,primary_key from system.tables where database='pf' and name='migration'
select * from system.parts where database='pf' and table='migration' limit 100
```

**Comment by shurshun** [comment]
I have the same problem:
```
2019/06/25 14:51:56 Create backup '2019-06-25T14-51-56'
2019/06/25 14:51:56 Freeze 'project.test'
2019/06/25 14:51:56   partition '201906'
2019/06/25 14:51:56 can't freeze partition '201906' on 'project.test' with: code: 368, message: std::bad_typeid
```

clickhouse-backup --version
```
Version:
Git Commit:	 12d508b8e718aba2e6a562abcc30e10540fe5482
Build Date:	 2019-06-17
```
ClickHouse version is 19.9.2.4

```
SELECT
    create_table_query,
    partition_key,
    primary_key
FROM system.tables
WHERE (database = 'project') AND (name = 'test')

Row 1:
──────
create_table_query: CREATE TABLE project.test (`event_time` DateTime, `event_date` Date DEFAULT toDate(event_time), `uuid` String) ENGINE = ReplicatedMergeTree('/clickhouse/tables/{shard}/test', '{replica}', event_date, event_date, 8192)
partition_key:      toYYYYMM(event_date)
primary_key:        event_date
```

```
SELECT *
FROM system.parts
WHERE (database = 'project') AND (table = 'test')
LIMIT 100

Row 1:
──────
partition:                             201906
name:                                  20190625_20190625_0_0_0
active:                                1
marks:                                 1
rows:                                  1
bytes_on_disk:                         163
data_compressed_bytes:                 113
data_uncompressed_bytes:               11
marks_bytes:                           48
modification_time:                     2019-06-25 14:51:20
remove_time:                           0000-00-00 00:00:00
refcount:                              1
min_date:                              2019-06-25
max_date:                              2019-06-25
min_time:                              0000-00-00 00:00:00
max_time:                              0000-00-00 00:00:00
partition_id:                          201906
min_block_number:                      0
max_block_number:                      0
level:                                 0
data_version:                          0
primary_key_bytes_in_memory:           2
primary_key_bytes_in_memory_allocated: 64
is_frozen:                             0
database:                              project
table:                                 test
engine:                                ReplicatedMergeTree
path:                                  /var/lib/clickhouse/data/project/test/20190625_20190625_0_0_0/
```

**Comment by AlexAkulov** [comment]
Thank you very much!
Since v19.4.5.35 freeze partitions by id was broken in ClickHouse. 
I found an alternative way and will fix it in the near future.

**Comment by AlexAkulov** [comment]
It must be fixed in v0.3.6

**Comment by MikeVL** [comment]
v0.3.5

```
11:37 root@clickhouse ➜  ~  /usr/local/bin/clickhouse-backup create
2019/06/27 11:37:39 Create backup '2019-06-27T08-37-39'
2019/06/27 11:37:39 Freeze 'default.nginx_logs'
2019/06/27 11:37:39   partition '201903'
2019/06/27 11:37:39 can't freeze partition '201903' on 'default.nginx_logs' with: code: 368, message: std::bad_typeid
```
